### DISCUSSION TITLE
How can I exclude the whole page from indexing?

### DISCUSSION THREAD

**Discussion by sergeysova** [Pagefind/pagefind#361] [Q&A]
Hello, thank you for the great tool!

I have a documentation website. Some pages have content, but another is just a redirect page. I want to exclude them from search results. How can I do this?

**Comment by bglw** [comment]
Hey @sergeysova 👋 

Currently the best way to achieve this is to utilize the `data-pagefind-body` tag ([docs](https://pagefind.app/docs/indexing/#limiting-what-content-is-indexed))

This tag should be added to the pages you _do_ want in your index — any remaining pages will be excluded automatically. 

Let me know if that works out :)

**Reply by Karlstens** [reply]
In my usage case, I'd like pagefind to search 99% of the site, with exception to a few generated pages that summarize articles (_that act as a site glossary, summary and index_). 

For such a site layout, I'd like to add a pagefind tag to the three pages to exclude them from search. Is this possible? 

Or alternatively, is there a config that where site path ignore terms can be specified? For example, I want pagefind to ignore the following;
```
.\help\glossary
.\help\indexing
.\help\sitemap
```

So effectively, the ability to add **\help\** to a pagefind blacklist.


```
.\help\glossary
.\help\indexing
.\help\sitemap
.\help\support
```

Or, an array such as [".\help\glossary", ".\help\indexing", ".\help\sitemap"] where the sub folder "support" is still needing to be indexed.
